src/plugins/merge.ts

```diff
@@ -41,20 +56,26 @@ export class Merge extends ManifestPlugin {
     const releaseData: ReleaseData[] = [];
     const labels = new Set<string>();
     let rawUpdates: Update[] = [];
+    let rootRelease: CandidateReleasePullRequest | null = null;
     for (const candidate of candidates) {
       const pullRequest = candidate.pullRequest;
       rawUpdates = rawUpdates.concat(...pullRequest.updates);
       for (const label of pullRequest.labels) {
         labels.add(label);
       }
       releaseData.push(...pullRequest.body.releaseData);
+      if (candidate.path === '.') {
+        rootRelease = candidate;
```

Comment:
Since we're already iterating over each candidate pr, if we find the root package, we'll remember it for later when we build the pr title